contrib/DEBIAN/manpages/zcashd.1

```diff
@@ -219,7 +219,7 @@ Fee (in BTC/kB) to add to transactions you send (default: 0.00)
 .HP
 \fB\-rescan\fR
 .IP
-Rescan the block chain for missing wallet transactions on startup
+Rescan the blockchain for missing wallet transactions on startup
```

Comment:
I intend to open a PR to revert these inconsistent changes. We use "block chain", as per the Satoshi whitepaper.